File: Java/프로그래머스/0/120912. 7의 개수/7의 개수.java
class Solution {
    public int solution(int[] array) {
        int count=0;
        for(int i=0;i<array.length ;i++){
            String a=Integer.toString(array[i]);
            for(int j=0;j<a.length();j++){
                if(a.charAt(j)=='7') count++;
            }
        }
        int answer = count;
        return answer;
    }
}

File: 프로그래머스/0/120912. 7의 개수/7의 개수.js
function solution(array) {
    var count=0;
    for(var i=0;i<array.length ;i++){
        var a=(String)(array[i]);
        for(var j=0;j<a.length;j++){
            if(a.charAt(j)=='7') count++;
        }
    }
    var answer = count;
    return answer;
}